# Exercise 39 - Attribute Error
import math

print(math.cos(1))